daemons/based/based_callbacks.c

```diff
@@ -275,7 +275,7 @@ cib_common_callback_worker(uint32_t id, uint32_t flags, xmlNode * op_request,
         }
         return;
 
-    } else if (pcmk__str_eq(op, T_CIB_NOTIFY, pcmk__str_none)) {
+    } else if (pcmk__str_eq(op, PCMK__VALUE_CIB_NOTIFY, pcmk__str_none)) {
```

Comment:
Really this should become some kind of OP or CMD constant when we unify those. Made a note for later.

Reply:
Notify types could get their own constant type too but that's less compelling. I'd be happy to keep those as VALUEs.